DeviceSimulation/SimulationTests.cs

```diff
@@ -1,14 +1,25 @@
-﻿using System.Net;
+﻿using System.IO;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Generic;
 using Helpers.Http;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Shared;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 using Xunit;
 
 namespace DeviceSimulation
 {
     public class SimulationTests
     {
         private readonly IHttpClient httpClient;
-        private const string DS_ADDRESS = "http://127.0.0.1:9003/v1";
-
+        private readonly RegistryManager registry;
+        private const string DS_ADDRESS = "http://localhost:9003/v1";
+        private const string IOTHUB_CONNECTION_STRING = "HostName=iothub-gi6bx.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=ncS4YgSWTp85ESqCpmkzN99WcWlRl0stMJTu3WfrR10=";
```

Comment:
Since this is shared with RM, how will delete scenario impact RM? Should we use different hub?

Reply:
We should get this value from Env variables. Refer here for example: https://github.com/Azure/azure-iot-device-simulation-dotnet-tests/blob/master/StorageAdapter/StatusTest.cs